### ISSUE TITLE
Mobile VR port?

### ISSUE THREAD

**Issue by poVoq** [REGoth-project/REGoth#355]
Might sound crazy, but Gothic on a GearVR, OcculusGo or Daydream headset might actually be a of of fun ;)

**Comment by frabert** [comment]
I don't see any particular technical reason this might not work, but it needs special equipment for development and that costs money... In the end, I'd say that the absolute priority is getting the engine right and working properly, then there's all the time of the world to add stuff I guess :)

**Comment by poVoq** [comment]
Yeah understandable. But most <5 year old smartphone support Google Cardboard these days, and that's 75% of the work needed if you use the latest Google VR SDK.
